redis-experiment-publisher-multi-server.py

```diff
@@ -0,0 +1,70 @@
+import redis
```

Comment:
Maybe? Perhaps we need a single server example that is equivalent to the original example _and_ the multi-server example to show that it can do either.

Reply:
Okay

Reply:
Done